steps/cli/oc.go

```diff
@@ -165,6 +166,32 @@ var _ = gauge.Step("Update addon config with resolverStepActions as <resolverSte
 	}
 })
 
+var _ = gauge.Step("Verify versioned ecosystem tasks", func() {
+	taskList := cmd.MustSucceed("oc", "get", "task", "-n", "openshift-pipelines").Stdout()
+	requiredTasks := []string{"buildah", "git-cli", "git-clone", "kn", "kn-apply", "maven", "openshift-client", "s2i-dotnet", "s2i-go", "s2i-java", "s2i-nodejs", "s2i-perl", "s2i-php", "s2i-python", "s2i-ruby", "skopeo-copy", "tkn"}
+	getRequiredVersion := os.Getenv("OSP_VERSION")
+	requiredVersion := strings.ReplaceAll(getRequiredVersion, ".", "-")
+	for _, task := range requiredTasks {
+		taskWithVersion := task + "-" + requiredVersion + "-0"
+		if !strings.Contains(taskList, taskWithVersion) {
+			testsuit.T.Errorf("Task %s not found in namespace openshift-pipelines", taskWithVersion)
+		}
+	}
+})
+
+var _ = gauge.Step("Verify versioned ecosystem step actions", func() {
+	stepActionList := cmd.MustSucceed("oc", "get", "stepaction", "-n", "openshift-pipelines").Stdout()
+	requiredStepActions := []string{"git-clone"}
```

Comment:
We introduced two new step actions in 5.0/1.18 but keep it as is so that we can cherry-pick this PR to both 1.18 and 1.17 branch. Then you create a new PR that will add new step actions.

Reply:
Ack

Reply:
Created PR https://github.com/openshift-pipelines/release-tests/pull/601